features/Gauges/GaugeList.tsx

```diff
@@ -20,6 +20,7 @@ import {
 import { JarApy } from "../../containers/Jars/useJarsWithAPY";
 import { useUniPairDayData } from "../../containers/Jars/useUniPairDayData";
 import { Jars } from "../../containers/Jars";
+import { NETWORK_NAMES } from "../../containers/config";
```

Comment:
You can undo all the changes in this file. Pickle on Polygon will never call this file. Ditto for `FarmList`